creepy/protocol/handshake.py

```diff
@@ -32,27 +37,40 @@ class HandshakeProtocol:
 
     def __init__(self, authorized_keys_path=None):
         self.salt = secrets.token_bytes(self.SALT_SIZE)
-        if authorized_keys_path is None:
+        self._bobs = {}
+        self._load_authorized_keys(authorized_keys_path)
+
+    def _add_bob(self, key):
+        key_hash = self.pubkey_digest(key, self.salt)
+        self._bobs[key_hash] = Bob(key)
+
+    def _load_authorized_keys(self, path):
+        """
+        Load keys from authorized_keys file and it's ssh folder.
+
+        Args:
```

Comment:
I forgot to mention that we follow [numpydoc's style guide](https://numpydoc.readthedocs.io/en/latest/format.html).
```suggestion
        Args:
```